pages/common/nix-env.md

```diff
@@ -2,13 +2,17 @@
 
 > Manipulate or query Nix user environments.
 
-- Show available package with name or without name:
+- List installed packages:
 
-`nix-env -qa {{pkg_name}}`
+`nix-env -q`
+
+- Query installed packages:
 
-- Show the status of available packages:
+`nix-env -q {{search_term}}`
 
-`nix-env -qas`
+- Query available packages:
+
+`nix-env -qa {{pkg_name}}`
```

Comment:
Should I change this to use `{{search_term}}` like the previous example, instead of `{{pkg_name}}`?

If you already know the exact name of a package (as suggested by `{{pkg_name}}` but not `{{search_term}}`), it wouldn't be very helpful to search for it.

Reply:
Agreed. It is more consistent, and gives more meaning to the term "query".